test/functional/apps/visualize/_area_chart.js

```diff
@@ -153,5 +155,85 @@ export default function ({ getService, getPageObjects }) {
       const sideEditorExists = await PageObjects.visualize.getSideEditorExists();
       expect(sideEditorExists).to.be(false);
     });
+
+    describe('switch between Y axis scale types', () => {
+      before(initAreaChart);
```

Comment:
why do we `initAreaChart` twice ? (before describe and before it in this describe)

Reply:
could we make all this tests go on without re-initing the charts ? this way the tests will run much faster. if one fails all the subsequent ones will fail as well, but i think that's not the problem as the first failing one already indicated issues with axis scaling.

Reply:
I've just followed what was already there in functional tests on vertical_bar_chart test.
I will remove all additional `before` calls and check if this still works correctly.

Reply:
@ppisljar I've take a better look at other tests code: I think there are two way to manage this:
- reintialize the chart (having a clean state on the test)
- go back to a clean state on the test like (reopen the saved chart or similar) because in the case of the areachart for example, the last test opens the visualization in embed mode. We need to go back to the editor before proceeding.
Reinitialization seems to be the clean way to work (and you are not dependent on the side effects produced by other tests (this will also allows you to avoid breaking tests when adding new tests without taking care where to put your test.
What do you think?